cmd/cassandra-writer/main.go

```diff
@@ -156,6 +162,21 @@ func newService(session *gocql.Session, logger logger.Logger) consumers.Consumer
 	return repo
 }
 
+func makeTransformer(cfg config, logger logger.Logger) transformers.Transformer {
+	switch strings.ToUpper(cfg.transformer) {
+	case "SENML":
+		logger.Info("Using SenML transformer")
+		return senml.New(cfg.contentType)
+	case "JSON":
```

Comment:
should this be a constant?

Reply:
It's used only here, so I'd say it's not necessary (it only adds a couple of extra lines of code).

Reply:
I agree with @mteodor 
Maybe we can export it from `/pkg/messaging`? Currently is used only here yes, but also in all writers and probably will be used in other places too. In general, it would be cool to call it `messaging.format.senml`  or similar.

Reply:
That makes sense, but it looks like it's related to  [this comment](https://github.com/mainflux/mainflux/pull/1331#discussion_r559023531). I'm not sure if we should put these constants in `pkg/messaging/transformer` and have implementation info on the same location where the interface is, or in a separate `configuration` package. We have a lot of repetitive code in `cmd` and we should probably fix this in a separate PR attached to issue #1286.

Reply:
@dusanb94 Yea I agree about transformers, but I don't think it's useful only for configuration, either related to transformers. 
IMHO it's a simple string, a format type definition that we support (in your PR used for config and related to transformers yes, but not neccearly), It actually belongs to messaging in general (it defines message format type). That's why I see it in `/pkg/messaging` as exported available format, currently a type string, easily changeable to any other format.